shared/settings/landing/index.desktop.js

```diff
@@ -278,6 +278,26 @@ function AccountEmail({
   )
 }
 
+function AccountFirstEmail({onChangeEmail}: {onChangeEmail: () => void}) {
+  return (
+    <Box
+      style={{
+        ...globalStyles.flexBoxRow,
+        alignItems: 'center',
+        justifyContent: 'space-between',
+        minHeight: ROW_HEIGHT,
+      }}
+    >
+      <Text type="Body">
+        No e-mail set. You should{' '}
+        <Text type="Body" style={{color: globalColors.blue}} link={true} onClick={onChangeEmail}>
```

Comment:
`type="BodyPrimaryLink"` and no need for `style` or `link` props. `Text` doesn't use `link` anymore (I don't think it has for 1-2yrs 😮), we should update the rest of this file at some point.